[2022-08-19 00:36:11,731] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: docker_dag.print_current_date manual__2022-08-19T00:36:04.970417+00:00 [queued]>
[2022-08-19 00:36:11,766] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: docker_dag.print_current_date manual__2022-08-19T00:36:04.970417+00:00 [queued]>
[2022-08-19 00:36:11,774] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 00:36:11,776] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-08-19 00:36:11,780] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 00:36:11,832] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): print_current_date> on 2022-08-19 00:36:04.970417+00:00
[2022-08-19 00:36:11,845] {standard_task_runner.py:52} INFO - Started process 336 to run task
[2022-08-19 00:36:11,857] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 'print_current_date', 'manual__2022-08-19T00:36:04.970417+00:00', '--job-id', '590', '--raw', '--subdir', 'DAGS_FOLDER/docker_dag.py', '--cfg-path', '/tmp/tmpbad609xp', '--error-file', '/tmp/tmpnq0mllt2']
[2022-08-19 00:36:11,861] {standard_task_runner.py:80} INFO - Job 590: Subtask print_current_date
[2022-08-19 00:36:12,063] {task_command.py:369} INFO - Running <TaskInstance: docker_dag.print_current_date manual__2022-08-19T00:36:04.970417+00:00 [running]> on host fa66a233a56d
[2022-08-19 00:36:12,303] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=docker_dag
AIRFLOW_CTX_TASK_ID=print_current_date
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T00:36:04.970417+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-19T00:36:04.970417+00:00
[2022-08-19 00:36:12,310] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-19 00:36:12,313] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-08-19 00:36:12,359] {subprocess.py:85} INFO - Output:
[2022-08-19 00:36:12,395] {subprocess.py:92} INFO - Fri Aug 19 00:36:12 UTC 2022
[2022-08-19 00:36:12,410] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-19 00:36:12,480] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=docker_dag, task_id=print_current_date, execution_date=20220819T003604, start_date=20220819T003611, end_date=20220819T003612
[2022-08-19 00:36:12,554] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 00:36:12,697] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
